src/adapter/src/optimize/mod.rs

```diff
@@ -137,6 +137,8 @@ where
 pub struct OptimizerConfig {
     /// The mode in which the optimizer runs.
     pub mode: OptimizeMode,
+    /// Enable fast path optimization.
+    pub enable_fast_path: bool,
```

Comment:
Note that this allows us to create optimizer that are explicitly configured to not attempt fast path optimization, not only in `EXPLAIN` but also in regular execution mode (might be useful for testing).